cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/ApplyController.java

```diff
@@ -43,9 +46,9 @@ public Integer call() throws Exception {
 
             ApplyLogic applyLogic = new ApplyLogic(cfOperations);
 
-            Log.info("Interpreting YAML file...");
+            log.info("Interpreting YAML file...");
```

Comment:
Does the ApplyApplicationCommand need its own logger instance or is it fine if it uses the instance of ApplyController? (This question is also related to Controller classes in general)

Reply:
I'd do it as fine-grainedly as possible, now that we have the chance to do so. That's the utter point of using a logger factory.